consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/CMCConsumerGateway.java

```diff
@@ -0,0 +1,130 @@
+                              XmlConverter xmlToObject,
+                              XmlElementValueReader xmlElementValueReader) {
+        this.breathingSpace = breathingSpace;
+        this.judgementService = judgementService;
+        this.claimStatusUpdate = claimStatusUpdate;
+        this.claimDefences = claimDefences;
+        this.xmlToObject = xmlToObject;
+        this.xmlElementValueReader = xmlElementValueReader;
+    }
+
+    @Override
+    public void individualRequest(IIndividualRequest individualRequest,
+                                  long connectionTimeOut,
+                                  long receiveTimeOut) throws OutageException, TimeoutException {
+        LOGGER.debug("Invoke cmc target application service for individual request");
+        String sdtRequestReference = individualRequest.getSdtRequestReference();
+        String requestType = individualRequest.getRequestType();
+        String idamId = SdtContext.getContext().getCustomerIdamId();
+        String requestPayload = new String(individualRequest.getRequestPayload(), StandardCharsets.UTF_8);
+        try {
+            if (RequestType.JUDGMENT.getType().equals(requestType)) {
+                JudgementRequest judgementRequest = xmlToObject.convertXmlToObject(requestPayload,
+                                                                                   JudgementRequest.class);
+
+                JudgementResponse judgementResponse = judgementService.requestJudgment(idamId,
+                                                                                       sdtRequestReference,
+                                                                                       judgementRequest);
+                individualRequest.setTargetApplicationResponse(xmlToObject.convertObjectToXml(judgementResponse).getBytes(StandardCharsets.UTF_8));
+            } else if (RequestType.BREATHING_SPACE.getType().equals(requestType)) {
+                BreathingSpaceRequest request = xmlToObject.convertXmlToObject(requestPayload,
+                                                                               BreathingSpaceRequest.class);
+
+                BreathingSpaceResponse response = breathingSpace.breathingSpace(idamId, sdtRequestReference, request);
+                individualRequest.setRequestStatus(response.getProcessingStatus().name());
+            } else if (RequestType.CLAIM_STATUS_UPDATE.getType().equals(requestType)) {
+                ClaimStatusUpdateRequest request = xmlToObject.convertXmlToObject(requestPayload,
+                                                                                  ClaimStatusUpdateRequest.class);
+                ClaimStatusUpdateResponse response = claimStatusUpdate.claimStatusUpdate(idamId, sdtRequestReference, request);
+                individualRequest.setRequestStatus(response.getProcessingStatus().name());
+            }
```

Comment:
Should this have an else part?  Could just log that a non-matching request type was specified.